bedrock/firefox/templates/firefox/fights-for-you.de.html

```diff
@@ -109,13 +110,34 @@
 </div>
 
 <div id="fffy-video-modal" class="mzp-u-modal-content">
-    <div class="ytcontainer-video">
-      <div class="video-container">
-        <div id="fffy-video-player"></div>
-      </div>
+  <div class="ytcontainer-video">
+    <div class="video-container">
+      <div id="fffy-video-player"></div>
     </div>
+  </div>
 </div>
 
+<aside id="fffy-focus-modal" class="mzp-u-modal-content">
+  <h3 class="c-modal-title">Bitte QR-Code scannen, um loszulegen.</h3>
+
+  <div class="modal-content-wrapper">
+    <div class="qr-code-wrapper">
+      <img src="{{ static('img/firefox/fights-for-you/qrcode-klar.png') }}" alt="" width="300" height="300">
+    </div>
+
+    <ul class="mobile-store-buttons">
+      <li class="android">
+        {{ google_play_button(href='https://app.adjust.com/rtnpf1x') }}
+      </li>
+      <li class="ios">
+        <a href="https://app.adjust.com/rtnpf1x" data-link-type="download" data-download-os="iOS">
```

Comment:
If I click this link on desktop it takes me to the Play Store and not the App Store